settings.gradle

```diff
@@ -1,4 +1,4 @@
-rootProject.name = 'parent'
+rootProject.name = 'robolectric-parent'
```

Comment:
Could we keep to use `parent`?

Reply:
I think `parent` here is not specific, it's misleading in IDEA with other projects named `parent`.

Reply:
We can discuss it at new discussion, instead of this PR.

Reply:
Feel free to open a new discussion to discuss which name is better than current parent for Robolectric top project.

Reply:
Reverted, will discuss it in a new topic.